.storybook/util/filterTokens.ts

```diff
@@ -1,10 +1,19 @@
 import tokens from '../data/tokens.json';
 
+const recurseToPrimaryValue: (value: string) => string = (value) => {
```

Comment:
not sure why typing like this is necessary 🤔 
![image](https://github.com/chanzuckerberg/edu-design-system/assets/86632227/a2c9bd1f-e7ef-43ae-9404-2dd4116f8e8a)